### DISCUSSION TITLE
Add code for ShowHexId setting.

### DISCUSSION THREAD

**Discussion by BHNHAT** [winsiderss/systeminformer#1567] [General Discussion]
As the title says. The code for ShowHexId is gone after the big update.

**Comment by dmex** [comment]
`ShowHexId` is a legacy setting that was replaced with new columns ending with `(hex)`. You can have both columns simultaneously instead of having to choose between one or the other 👍

**Comment by BHNHAT** [comment]
Ok, I see it. Thanks for letting me know.
